vendor/engines/sanger_sequencing/app/models/sanger_sequencing/submission.rb

```diff
@@ -22,9 +22,15 @@ class Submission < ActiveRecord::Base
     alias purchased_at ordered_at
 
     scope :purchased, -> { joins(:order).merge(Order.purchased.order(ordered_at: :desc)) }
-    scope :ready_for_batch, -> { purchased.merge(OrderDetail.new_or_inprocess).where(batch_id: nil) }
     scope :for_facility, ->(facility) { where(orders: { facility_id: facility.id }) }
 
+    BATCHABLE_STATES = %w(new inprocess complete).freeze
```

Comment:
What happens to the UI if there are a ton of completed submissions? Will that be messy/make it difficult to find the submissions that haven't been run yet?